webapi/src/main/java/org/sakaiproject/webapi/controllers/CalendarController.java

```diff
@@ -53,34 +52,38 @@
 
 import lombok.extern.slf4j.Slf4j;
 
-/**
- */
 @Slf4j
 @RestController
 public class CalendarController extends AbstractSakaiApiController {
 
-	@Resource
-	private CalendarService calendarService;
+    private final static int DEFAULT_DAYS = 60;
+
+    @Autowired
+    private CalendarService calendarService;
+
+    @Autowired
+    private ContentHostingService contentHostingService;
 
-	@Resource
-	private ContentHostingService contentHostingService;
+    @Autowired
+    @Qualifier("org.sakaiproject.coursemanagement.api.CourseManagementService")
+    private CourseManagementService courseManagementService;
```

Comment:
I think you're not using this here anymore, you can remove this definition and the import.